app/migrations/Version20230601082815.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+
+final class Version20230601082815 extends PreUpAssertionMigration
+{
+    protected function preUpAssertions(): void
+    {
+        $this->skipAssertion(function (Schema $schema) {
+            $sql         = sprintf('select category_id from %s%s where category_id is null', $this->prefix, 'beefree_rows');
```

Comment:
This migration probably doesn't make sense for the community, right? They don't have the Beefree builder